src/main.ts

```diff
@@ -327,31 +326,18 @@ ipcMain.on('browser:stop', async () => {
 ipcMain.handle('script:select', async (event) => {
   console.info('script:select event received')
   const browserWindow = browserWindowFromEvent(event)
-
   const scriptPath = await showScriptSelectDialog(browserWindow)
-  console.info(`selected script: ${scriptPath}`)
-
-  if (!scriptPath) return
-
-  const fileHandle = await open(scriptPath, 'r')
-  try {
-    const script = await fileHandle?.readFile({ encoding: 'utf-8' })
 
-    return { path: scriptPath, content: script }
-  } finally {
-    await fileHandle?.close()
-  }
+  return scriptPath
```

Comment:
We only need the path anyway, because there's a route redirect on selection and the actual loading is handled after the redirection